examples/tutorials/analysis-time/Variability_estimation.py

```diff
@@ -0,0 +1,193 @@
+flux_err = lc_1d.flux_err.quantity
+
+f_mean = np.mean(flux)
+f_mean_err = np.mean(flux_err)
+
+f_max = flux.max()
+f_max_err = flux_err[flux.argmax()]
+f_min = flux.min()
+f_min_err = flux_err[flux.argmin()]
+
+amplitude_maximum_variation = (f_max - f_max_err) - (f_min + f_min_err)
+
+amplitude_maximum_significance = amplitude_maximum_variation / np.sqrt(
+    f_max_err**2 + f_min_err**2
+)
+
+print(amplitude_maximum_significance)
+
+# There are other methods based on the peak-to-trough difference to assess the variability in a lightcurve.
+# Here we present as example the relative variability amplitude
+# ([reference paper](https://iopscience.iop.org/article/10.1086/497430)):
+
+relative_variability_amplitude = (f_max - f_min) / (f_max + f_min)
+
+relative_variability_error = (
+    2
+    * np.sqrt((f_max * f_min_err) ** 2 + (f_min * f_max_err) ** 2)
+    / (f_max + f_min) ** 2
+)
+
+relative_variability_significance = (
+    relative_variability_amplitude / relative_variability_error
+)
+
+print(relative_variability_significance)
+
+# And the variability amplitude ([reference paper](https://ui.adsabs.harvard.edu/abs/1996A%26A...305...42H/abstract)):
+
+variability_amplitude = np.sqrt((f_max - f_min) ** 2 - 2 * f_mean_err**2)
+
+variability_amplitude_100 = 100 * variability_amplitude / f_mean
+
+variability_amplitude_error = (
+    100
+    * ((f_max - f_min) / (f_mean * variability_amplitude_100 / 100))
+    * np.sqrt(
+        (f_max_err / f_mean) ** 2
+        + (f_min_err / f_mean) ** 2
+        + ((np.std(flux, ddof=1) / np.sqrt(len(flux))) / (f_max - f_mean)) ** 2
+        * (variability_amplitude_100 / 100) ** 4
+    )
+)
+
+variability_amplitude_significance = (
+    variability_amplitude_100 / variability_amplitude_error
+)
+
+print(variability_amplitude_significance)
+
+######################################################################
+#  Fractional excess variance, point-to-point fractional variance and doubling/halving time
+# ----------------------------------------------
+# The fractional excess variance ([reference
+# paper](https://ui.adsabs.harvard.edu/abs/2003MNRAS.345.1271V/abstract)) is a simple but effective method to assess
+# the significance of a time variability feature in an object, for example an AGN flare.
+# It is important to note that it requires gaussian errors to be applicable.
+# The excess variance computation is implemented in `~gammapy.estimators.utils`.
+# A similar estimator is the point-to-point fractional variance, which samples the lightcurve with smaller time granularity.
+# In general, the point-to-point fractional variance being higher than the fractional excess variance is indicative of the presence of very short timescale variability.
+#
+# The doubling and halving time of the light curve can also be computed.
+# This provides information on the shape of the variability feature.
+
+fvar_table = compute_lightcurve_fvar(lc_1d)
+print(fvar_table)
+
+fpp_table = compute_lightcurve_fpp(lc_1d)
```

Comment:
Maybe for clarity you can have one cell with each quantity estimated so that you can precisely explain what is done by each function and give proper reference before using them.